quick-protobuf/examples/pb_rs/data_types.rs

```diff
@@ -112,26 +112,63 @@ pub struct FooMessage<'a> {
     pub test_oneof: mod_FooMessage::OneOftest_oneof<'a>,
 }
 
+impl<'a> FooMessage<'a> {
+    pub const DEFAULT_f_sint64: i64 = 4i64;
+    pub const DEFAULT_f_bool: bool = true;
+    pub const DEFAULT_f_fixed32: u32 = 0u32;
+}
+
+
+impl<'a> Default for FooMessage<'a> {
+    fn default() -> Self {
+        Self {
+            f_int32: None,
+            f_int64: None,
+            f_uint32: None,
+            f_uint64: None,
+            f_sint32: None,
+            f_sint64: None,
+            f_bool: None,
+            f_FooEnum: None,
+            f_fixed64: None,
+            f_sfixed64: None,
+            f_fixed32: None,
+            f_sfixed32: None,
+            f_double: None,
+            f_float: None,
+            f_bytes: None,
+            f_string: None,
+            f_self_message: None,
+            f_bar_message: None,
+            f_repeated_int32: Vec::new(),
+            f_repeated_packed_int32: Vec::new(),
+            f_repeated_packed_float: PackedFixed::from(Vec::new()),
+            f_imported: None,
+            f_baz: None,
+            f_nested: None,
+            f_nested_enum: None,
+            f_map: HashMap::new(),
+            test_oneof: mod_FooMessage::OneOftest_oneof::None,
+        }
+    }
+}
```

Comment:
Why is this one not derived?

Reply:
Oversight in the `impl Default` system; have fixed it😅 thanks for catching